dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaManagerConnector.java

```diff
@@ -82,6 +87,12 @@ public void addTasksTo(@Nonnull List<? super Task<?>> out, @Nonnull ConnectorArg
     out.add(new ClouderaCMFHostsTask());
     out.add(new ClouderaAPIHostsTask());
     out.add(new ClouderaServicesTask());
+    for (int deltaInDays : Arrays.asList(1, 7, 30, 90)) {
+      out.add(
+          new ClouderaClusterCPUChartTask(
+              ZonedDateTime.of(LocalDateTime.now().minusDays(deltaInDays), ZoneId.of("UTC")),
+              deltaInDays > 1 ? TimeSeriesAggregation.DAILY : TimeSeriesAggregation.HOURLY));
+    }
```

Comment:
This init is smart but require some time to understand what is happens here. It should simple declaration/enumeration of tasks which should be compleated.
Something like that:
```
out.add(new ClouderaClusterCPUChartTask(1, TimeSeriesAggregation.HOURLY));
out.add(new ClouderaClusterCPUChartTask(7, TimeSeriesAggregation.DAILY));
out.add(new ClouderaClusterCPUChartTask(30, TimeSeriesAggregation.DAILY));
out.add(new ClouderaClusterCPUChartTask(90, TimeSeriesAggregation.DAILY));
```

Reply:
Fixed.